gradle.properties

```diff
@@ -3,6 +3,9 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 org.gradle.configureondemand=true
 org.gradle.caching=true
 org.gradle.parallel=true
+# configuration-cache
+org.gradle.unsafe.configuration-cache=true
+org.gradle.unsafe.configuration-cache-problems=warn
```

Comment:
@masah517 
Thanks for trying!!

`configuration-cache-problems=warn` causes a problem that the error does not occur when the configuration cache is ineffective.
Could you try enable `configuration cache` without `problems=warn` option ?

FYI: AGP and MPP gradle plugin has already support configuration cache 😉
(maybe we need to migration convention plugin to support configuration cache)

Reply:
@RyuNen344 
Will try as advised! Thank you

Reply:
@RyuNen344 
Hi there!
It seems like the build keeps failing even though I remove `problems=warn` option.

May I please have support on what you mean by 
> maybe we need to migration convention plugin to support configuration cache)

Thank you

Reply:
Yes, of course. Enabling configuration caches properly needs 4 steps.

- Turn the problem level to warn.
- Support confiugration caches for owned custom plugins including convention plugins by following https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:testing.
- Turn it back to error.
- Until the build passes, specify Task#notCompatibleWithConfigurationCache for 3rd party tasks that haven't supported caches.

Please note that it's not the beat approach that just turns configuration cache problems into warn level. The option helps developers to support caches gradually but not a fisrt choice for enabling caches by plugin users. If you have to do it, ` org.gradle.configuration-cache.max-problems=<small mumber>` must to be set to control your projects.